libpod/runtime_ctr.go

```diff
@@ -263,3 +267,22 @@ func (r *Runtime) GetContainersByList(containers []string) ([]*Container, error)
 	}
 	return ctrs, nil
 }
+
+// GetLatestContainer is a helper function for GetContainers which
```

Comment:
Nit: Don't mention that it's a helper function, just say that it retrieves the last container created

Reply:
ok?